app/lib/common/toolkit/errors/inline_error_button.dart

```diff
@@ -70,26 +65,22 @@ class ActerInlineErrorButton extends StatelessWidget {
     this.includeBugReportButton = true,
   });
 
-  Widget _buildWithIcon(BuildContext context) {
+  Widget _buildWithIcon(BuildContext context, Icon icn) {
     return IconButton(
-      icon: icon!,
-      onPressed: () async {
-        await ActerErrorDialog.show(
-          context: context,
-          error: error,
-          stack: stack,
-          title: dialogTitle,
-          text: text,
-          textBuilder: textBuilder,
-          onRetryTap: onRetryTap != null
-              ? () {
-                  onRetryTap!();
-                  Navigator.pop(context);
-                }
-              : null,
-          includeBugReportButton: includeBugReportButton,
-        );
-      },
+      icon: icn,
```

Comment:
we can spell it out: `Icon icon`. Generally in dart or rust abbreviating short words isn't really that great.

Reply:
Frankly, I renamed it from `icon` to `icn` in order to avoid confusing with `this.icon`.

Reply:
well, okay then.